tfjs-backend-webgpu/src/kernels/pool_filtersizeone_webgpu.ts

```diff
@@ -38,7 +38,7 @@ export class MaxPoolWithFilterSizeEqualsOneProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
         this.dispatchLayout, this.outputShape, this.workGroupSize);
 
-    this.shaderKey = 'maxpoolv2';
+    this.shaderKey = 'poolv2';
```

Comment:
Prefer to use a meaningful key name to keep consistent with others. For example `poolWithFilterSizeEqualsOne`